### PR TITLE
Implement `getFreeMonitoringStatus` stub

### LINKED ISSUE #745: Implement `getFreeMonitoringStatus` command (stubs)
Implement `getFreeMonitoringStatus` command stub that would always return "monitoring is not enabled". See:
* https://www.mongodb.com/docs/manual/reference/command/getFreeMonitoringStatus/
* https://www.mongodb.com/docs/manual/administration/free-monitoring/

We ~probably~ definitely will need to change MongoDB configuration to disable free monitoring, so the state is "disabled", not "undecided". That would allow us to write a simpler integration test. See #748.

### PR DESCRIPTION
**Pull request by noisersup**
Closes #745.

# Checklist

* [x] Tests are added for new functionality or fixed bugs.
* [x] `task all` passes.

See CONTRIBUTING.md for more details.

### CODE REVIEW

**Comment by noisersup** [comment]
> I'm not sure why `task lint` return an error on my machine.
> 
> ```
> internal/handlers/common/msg_hostinfo.go:65:42: directive `//nolint:staticcheck // good enough for GOOS` is unused for linter "staticcheck"
> (nolintlint)
>                                 "type", strings.Title(runtime.GOOS), //nolint:staticcheck // good enough for GOOS
>                                                                      ^
> task: Failed to run task "lint-golangci-lint": exit status 1
> ```
> 
> I didn't even change `internal/handlers/common/msg_hostinfo.go` file.

Fixed with `bin/golangci-lint cache clean`

**Comment by AlekSi** [comment]
#756

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: internal/handlers/common/msg_getfreemonitoringstatus.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	"github.com/FerretDB/FerretDB/internal/wire"
+)
+
+// MsgGetFreeMonitoringStatus is a common implementation of the getFreeMonitoringStatus command.
+func MsgGetFreeMonitoringStatus(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
+	var reply wire.OpMsg
+	err := reply.SetSections(wire.OpMsgSection{
+		Documents: []*types.Document{must.NotFail(types.NewDocument(
+			"state", "disabled",
+			"message", "monitoring is not enabled", // Should I include that?
```
[the remark this answers is not in the document]
Comment (noisersup) [reply internal/handlers/common/msg_getfreemonitoringstatus.go:32]: whoops, sorry for overlooking

File: internal/handlers/common/msg_listcommands.go [hunk 2]
```diff
@@ -107,6 +107,10 @@ var Commands = map[string]command{
 		Help:    "Returns a summary of all runtime and configuration options.",
 		Handler: (handlers.Interface).MsgGetCmdLineOpts,
 	},
+	"getFreeMonitoringStatus": {
+		Help:    "MsgGetFreeMonitoringStatus returns the status of free Cloud monitoring",
```
Comment (AlekSi) [inline internal/handlers/common/msg_listcommands.go:111]:
```suggestion
		Help:    "MsgGetFreeMonitoringStatus returns the status of free Cloud monitoring.",
```

**Review by AlekSi — DISMISSED** [review]

**Review by AlekSi — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
LGTM

File: integration/commands_freemonitoring_test.go [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	for k, item := range expected {
+		assert.Contains(t, keys, k)
+		assert.IsType(t, item, m[k])
+
+		if it, ok := item.(primitive.D); ok {
+			z := m[k].(primitive.D)
+			AssertEqualDocuments(t, it, z)
+		} else {
+			assert.Equal(t, m[k], item)
+		}
```
Comment (seeforschauer) [inline integration/commands_freemonitoring_test.go:53]: I would suggest here reducing complexity.
```go
                        AssertEqualDocuments(t, it, z)
                        continue
		} 
		assert.Equal(t, m[k], item)
```

File: internal/handlers/common/msg_listcommands.go [hunk 4]
```diff
@@ -107,6 +107,10 @@ var Commands = map[string]command{
 		Help:    "Returns a summary of all runtime and configuration options.",
 		Handler: (handlers.Interface).MsgGetCmdLineOpts,
 	},
+	"getFreeMonitoringStatus": {
+		Help:    "MsgGetFreeMonitoringStatus returns the status of free Cloud monitoring.",
```
Comment (seeforschauer) [inline internal/handlers/common/msg_listcommands.go:111]: Doesn't it has to be in a small letter? 
```suggestion
		Help:    "MsgGetFreeMonitoringStatus returns the status of free cloud monitoring.",
```

File: internal/handlers/handlers.go [hunk 5]
```diff
@@ -84,6 +84,9 @@ type Interface interface {
 	// MsgGetCmdLineOpts returns a summary of all runtime and configuration options.
 	MsgGetCmdLineOpts(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
 
+	// MsgGetFreeMonitoringStatus returns the status of free Cloud monitoring.
```
Comment (seeforschauer) [inline internal/handlers/handlers.go:87]: same here

**Review by seeforschauer — APPROVED** [review]

**Comment by noisersup** [comment]
> wait, tests not passing :)

That's weird, I'm sure that I have run `test all` before pushing.

**Comment by noisersup** [comment]
there are some [authentication problems](https://github.com/FerretDB/FerretDB/runs/7003515496?check_suite_focus=true#step:7:17) in tests I guess

**Comment by AlekSi** [comment]
That was a temporary GitHub hiccup

**Review by seeforschauer — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]

**Pull request merged**